File: customer-service/src/main/java/com/orbitbank/customerservice/model/Customer.java
package com.orbitbank.customerservice.model;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.index.Indexed;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "customer")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Customer {
    @Id
    @Indexed(unique = true)
    private String customerId;
    private String firstName;
    private String lastName;
    private String email;
    private String phone;
    private String dob;
    private String city;
    private String district;
    private String state;
    private String country;
    private String panNo;
    private String aadhaarNo;
    private String accNo;
    private String accountStatus="active";


}

File: bank-service/src/main/kotlin/com/orbitbank/bankservice/model/Customer.kt
package com.orbitbank.bankservice.model

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor
import java.util.Date
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Data
@AllArgsConstructor
@NoArgsConstructor
class Customer (
    @Id
    @GeneratedValue
    var id: Int,
    var FirstName: String,
    var LastName: String,
    var DOB: Date,
    var City: String,
    var District: String,
    var State: String,
    var Country: String,
    var PanNo: String,
    var AadhaarNo: Long,
    var BankAcc: Long,
)